src/proto/mgmt/pool.proto

```diff
@@ -49,11 +49,9 @@ message CreatePoolResp {
 
 // DestroyPoolReq supplies pool identifier and force flag.
 message DestroyPoolReq {
-	string uuid = 1;
-	bool force = 2;
+	string uuid = 1; // uuid of pool to destroy
+	string sys = 2; // DAOS system identifier
+	bool force = 3; // destroy regardless of active connections
 }
 
-// DestroyPoolResp returns status of processed request.
-message DestroyPoolResp {
-	DaosRequestStatus status = 1;
-}
\ No newline at end of file
+// DestroyPoolResp is identical to DaosResp.
```

Comment:
The previous DestroyPoolResp is better, especially for a client-server RPC whose compatibility may be our concern.